src/net/http.js

```diff
@@ -169,10 +165,9 @@ pc.extend(pc, function () {
          * @param {Document|Object} [options.postdata] Data to send in the body of the request.
          * Some content types are handled automatically, If postdata is an XML Document it is handled, if the Content-Type header is set to 'application/json' then
          * the postdata is JSON stringified, otherwise by default the data is sent as form-urlencoded
-         * @param {Object} [options.cache] If false, then add a timestamp to the request to prevent caching
          * @param {Function} callback The callback used when the response has retured. Passed (err, data) where data is the response (format depends on response type, text, Object, ArrayBuffer, XML) and err is the error code.
          */
-        request: function (method, url, options, callback) {
+        request(method, url, options, callback) {
```

Comment:
Is this right? 'request' is a property defined as a function. Why have you dropped the ': function'?

Reply:
Sorry about that, it is a mistake. I think the wires got crossed between my TypeScript declaration file and the source code.